app/src/main/res/drawable/thumb_ibm.xml

```diff
@@ -0,0 +1,14 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="1000dp"
+    android:height="400dp"
+    android:viewportWidth="1000"
+    android:viewportHeight="400">
```

Comment:
As above, the image should be at most 128x128

Reply:
done, see new push